oio/cli/cluster/cluster.py

```diff
@@ -89,32 +93,34 @@ def _list_services(self, parsed_args):
                 continue
             for srv in data:
                 tags = srv['tags']
-                location = tags.get('tag.loc', 'n/a')
-                slots = tags.get('tag.slots', 'n/a')
-                volume = tags.get('tag.vol', 'n/a')
-                service_id = tags.get('tag.service_id', 'n/a')
+                location = tags.pop('tag.loc', 'n/a')
+                slots = tags.pop('tag.slots', 'n/a')
+                volume = tags.pop('tag.vol', 'n/a')
+                service_id = tags.pop('tag.service_id', 'n/a')
                 addr = srv['addr']
-                locked = boolean_value(tags.get('tag.lock'), False)
-                up = tags.get('tag.up', 'n/a')
+                locked = boolean_value(tags.pop('tag.lock'), False)
```

Comment:
We should also set a default value as second argument to pop, or it may raise KeyError.